src/besselj.jl

```diff
@@ -115,33 +218,28 @@ function besselj1(x::Float64)
     isinf(x) && return zero(x)
 
     if x <= 5.0
```

Comment:
This branch should be less than ~26? Or wherever the minimax was done up to.

Reply:
good catch.